extensions/container-image/deployment/src/main/resources/dev-ui/container-image/build-container-component.js

```diff
@@ -0,0 +1,144 @@
+import {LitElement, html, css, render} from 'lit';
+import {JsonRpc} from 'jsonrpc';
+import '@vaadin/icon';
+import '@vaadin/button';
+import {until} from 'lit/directives/until.js';
+import '@vaadin/grid';
+import '@vaadin/grid/vaadin-grid-sort-column.js';
+import {builderTypes} from 'container-image-data';
+import '@vaadin/text-field';
+import '@vaadin/text-area';
+import '@vaadin/form-layout';
+import '@vaadin/progress-bar';
+import '@vaadin/checkbox';
+import '@vaadin/select';
+import '@vaadin/item';
+import '@vaadin/list-box';
+
+
+export class BuildContainerComponent extends LitElement {
+
+    jsonRpc = new JsonRpc("ContainerBuilder");
```

Comment:
Use `Container Image`

Reply:
I would avoid spaces, I went for ContainerImage

Reply:
What I did with using this in a path is to do toLowerCase and replace space with -. Maybe this is better ? Again, I want to be able to do this automatically in the future from the UI, so need some pattern to know how to get the jsonrpc. Also, spaces is ok :)